examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<ol><li>Add mode details about what are you going to do here and what is the difference with the previous section</li><li>Also add subsections here</li><li class="ql-indent-1">Prepare data in etna hierarchical long format</li><li class="ql-indent-1">Convert data to etna wide format with <code>to_hierarchical_dataset</code></li><li class="ql-indent-1">Create the hierarchical dataset </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>